packages/web-components/src/components/cbp-structured-list/cbp-structured-list.stories.tsx

```diff
@@ -409,28 +409,206 @@ StructuredListMediaCalendar.argTypes = {
     control: 'object',
   },
 }
-StructuredListMediaCalendar.args = { //todo: update text in samples to match calendar text
+StructuredListMediaCalendar.args = {
   showHeader: false,
   listItems: [
     {
-      content: "<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'><div class='cbp-structured-list-calendar-item'><span class='cbp-structured-list-calendar-item-month'>apr</span><span class='cbp-structured-list-calendar-item-day'>31</span></div><div><cbp-typography tag='p'><cbp-icon name='arrow-right' color='var(--cbp-link-color)' size='1rem'></cbp-icon><cbp-link href='#' target='_self' style='margin-inline-start: 0.5rem'>Find the Hidden Beetle Training</cbp-link></cbp-typography><cbp-typography tag='p'>This training session will help you find those pesky hiding beetles. Inside boxes, on top of containers, or in someone's pocket, these bugs don't stand a chance of getting to America's sweet, delicious forests</cbp-typography><cbp-typography tag='p'><cbp-icon name='globe' size='1rem'></cbp-icon> <i>https://www.cbp.dhs.gov/news/beetles</i></cbp-typography></div><cbp-grid>",
+      content: `<cbp-grid grid-template-columns='1fr 5fr' gap='1rem'>
+                  <cbp-flex
+                  sx='{
+                        "width":"6.25rem",
+                        "height":"6.25rem",
+                        "display":"flex",
```

Comment:
you don't need display: flex since you are using the flex component. flex-direction and align-items are implemented as props on this component, so use them there (better readability).

Reply:
update display, flex-direction, & align to props, removed the SX since width is being set on grid and I noticed some weirdness having the above set via prop the SX seemed to override those inline-styles instead of adding to